test/libsolidity/syntaxTests/string/invalid_utf8_sequence.sol

```diff
@@ -2,5 +2,5 @@ contract C {
     string s = unicode"�";
 }
 // ----
-// SyntaxError 8452: (28-38): Invalid UTF-8 sequence found
-// TypeError 7407: (28-38): Type literal_string (contains invalid UTF-8 sequence at position 0) is not implicitly convertible to expected type string storage ref.
+// SyntaxError 8452: (28-38): Invalid UTF-8 sequence found at position 0
```

Comment:
This extra check is there for even no-op expressions like `unicode"À"` so I guess we have to keep it.